app/javascript/packs/puerto-rico-guidance.ts

```diff
@@ -13,5 +17,30 @@ function onStateSelectionChange() {
   showOrHidePuertoRicoExtras(stateSelector?.value);
 }
 
+function onIdentityDocStateSelection() {
+  const stateSelectors = document.querySelectorAll<HTMLSelectElement>('.usa-select');
+  stateSelectors.forEach((stateSelector) => {
+    if (!stateSelector.id.includes('jurisdiction')) {
+      stateSelector.addEventListener('change', () =>
```

Comment:
Consider using the [`input` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) instead of the [`change` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) here. This makes it more likely that the text will be appropriately modified if the user has an extension automatically fill out the form.

Reply:
i'm going to opt to keep `change` as `input` fails the tests.